charts/icm-as/templates/_initContainer.tpl

```diff
@@ -4,7 +4,7 @@ Creates init-containers
 */}}
 {{- define "icm-as.initContainers" -}}
 initContainers:
-{{- if eq .Values.persistence.sites.type "local" }}
+{{- if eq .Values.persistence.sites.type "local_" }}
```

Comment:
intentional change? i see `persistence.sites.type: local` in the values but not `local_`